### PR TITLE
Add bibtex format citing and third-party references

### PR DESCRIPTION
**Pull request by KornbergFresnel**
add bibtex format citing and third-party references (benchmarking).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,3 +251,18 @@ $ python examples/single_agent.py scenarios/loop
 * If you wish to find binaries: **[SUMO Download Page](https://sumo.dlr.de/docs/Downloads.php)**
 * If you wish to compile from source see: **[SUMO Build Instructions](https://sumo.dlr.de/docs/Developer/Main.html#build_instructions)**
 * If you build from the git repository use: **[SUMO version 1.7.0](https://github.com/eclipse/sumo/commits/v1_7_0)** or higher
+
+### Citing SMARTS
+
+If you use SMARTS in your research, please cite the [paper](https://arxiv.org/abs/2010.09776). In BibTeX format:
+
+```bibtex
+@misc{zhou2020smarts,
```
Comment (liamchzh) [inline README.md:260]: `zhou2020smarts`? not sure whether it is an appropriate example.

Comment (JVillella) [reply README.md:260]: +1, what about something just like `@misc{smarts,`

Comment (KornbergFresnel) [reply README.md:260]: oh, I exported it from arXiv directly.

**Review by JVillella — APPROVED** [review]

**Review by liamchzh — APPROVED** [review]

**Comment by Gamenot** [comment]
I am merging this, there is a follow-up #80.

**Pull request merged**